src/main/java/org/example/Car.java

```diff
@@ -0,0 +1,69 @@
+package org.example;
+
```

Comment:
what does this class do? why these methods and not others? what are the key parameters?... some javadocs please